doc/source/admin/scaling.md

```diff
@@ -1,314 +1,570 @@
+of `job_conf.xml`. *Do not* define a default handler.
 
-This will let us manage these tasks more globally with the `supervisorctl` command line tool:
+For example, to have the 2nd mule in the three-mule job-handlers farm shown above handle the `test1` tool, you would set
+the following in `job_conf.xml` (irrelevant options are not shown):
 
-```console
-# supervisorctl status
-galaxy:handler0                  RUNNING   pid 7275, uptime 16:32:17
-galaxy:handler1                  RUNNING   pid 7276, uptime 16:32:17
-galaxy:uwsgi                     RUNNING   pid 7299, uptime 16:32:16
+```xml
+<job_conf>
+    <handlers>
+        <handler id="main.job-handlers.2" />
+    </handlers>
+    <tools>
+        <tool id="test1" handler="main.job-handlers.2" />
+    </tools>
+</job_conf>
 ```
 
+#### uWSGI + Webless job handling
 
-This command shows us the status of our jobs, and we can easily restart all of the processes at once by naming the group. Familiar commands like start and stop are also available.
+Define a `<handlers>` section in `job_conf.xml` defining the webless handlers you plan to start. In this case, unlike
+the uWSGI job handling strategies, you will need to define a default:
 
-```console
-# supervisorctl restart galaxy:
-galaxy:handler0: stopped
-galaxy:handler1: stopped
-galaxy:uwsgi: stopped
-galaxy:uwsgi: started
-galaxy:handler0: started
-galaxy:handler1: started
+```xml
+<job_conf>
+    <handlers default="handlers">
+        <handler id="handler1" tags="handlers" />
+        <handler id="handler2" tags="handlers" />
+        <handler id="handler3" />
+    </handlers>
+    <tools>
+        <tool id="test1" handler="handler3" />
+    </tools>
+</job_conf>
 ```
 
-### Transparent restart - Zerg Mode
+The definition of a default handler prevents uWSGI web workers from starting the Galaxy job handling code. `run.sh` will
+start the uWSGI process(es), but you will need to start the webless handler processes yourself. This is done on the
+command line like so:
 
-The standard uWSGI operation mode allows you to restart the Galaxy application while blocking client connections. Zerg Mode does away with the waiting by running a special Zerg Pool process, and connecting Zergling workers (aka Galaxy application processes) to the pool. As long as at least one is connected, requests can be served.
+```sh-session
+$ cd /srv/galaxy/server
+$ ./scripts/galaxy-main -c config/galaxy.yml --server-name handler0 --daemonize
+$ ./scripts/galaxy-main -c config/galaxy.yml --server-name handler1 --daemonize
+$ ./scripts/galaxy-main -c config/galaxy.yml --server-name handler2 --daemonize
+```
 
-See the [GCC2017 Admin Training session](https://github.com/galaxyproject/dagobah-training/blob/2017-montpellier/sessions/10-uwsgi/ex2-zerg-mode.md) on how to set this up.
+However, a better option to managing processes by hand is to use a process manager as documented in the [Starting and
+Stopping](#starting-and-stopping) section.
 
-## Proxy Server
```

Comment:
@natefoo Are you planning to move the instructions for nginx and Apache configuration for uWSGI to doc/source/admin/special_topics/nginx.md and https://github.com/jmchilton/galaxy/blob/35b1e2c9e7ce349f8cc37d363bb637b96a37266b/doc/source/admin/special_topics/apache.md respectively?

Reply:
You probably saw this, but for anyone else wondering, yes, this was addressed in the nginx/Apache docs.

Reply:
Yes, thanks!